app/Resources/TwigBundle/views/Exception/error500.html.twig

```diff
@@ -9,10 +9,10 @@
 
             <div class="error-content">
                 <h3><i class="fa fa-warning text-red"></i> T'as tout cassé !</h3>
-
                 <p>
-                    Si le problème persiste, tu nous envoyer un petit rapport d'erreur à <a href="mailto:dsi@N7consulting.fr">dsi@N7consulting.fr</a> <br>
-                    Explique bien le contexte et mets des petites images et tout et tout ;)
+                    Si le problème persiste, tu peux nous envoyer un petit rapport d'erreur à <a href="mailto:{{ technical_to }}">{{ technical_to }}</a><br>
+                    Pour que le problème soit rapidement pris en compte et traité, le mieux serait de nous faire un rapport directement sur <a href="https://github.com/n7consulting/Incipio/issues/new">GitHub</a></br>
```

Comment:
`<br>` or `<br />` but not `</br>`

Problem common to the different error views